ToDo-GUI/todo.py

```diff
@@ -0,0 +1,94 @@
+#ToDo GUI
+import tkinter
+from tkinter import END, ANCHOR
+count=0
+
+
+#Defining root window
+root = tkinter.Tk()
+root.title('ToDo-GUI')
+root.iconbitmap('todo.ico')
+root.geometry('400x400')
+root.resizable(0,0)
+
+#Define fonts and colors
+my_font = ('Times New Roman', 12)
+root_color = 'green'
+button_color = '#e2cff4'
+root.config(bg=root_color)
+
+#Define functions
+def add_item():
+    global count
+    count=count+1
+    my_listbox.insert(END,str(count)+") "+ list_entry.get())
+    list_entry.delete(0, END)
+
+
+def remove_item():    
+    my_listbox.delete(ANCHOR)
+
+
+def clear_list():   
+    global count 
+    my_listbox.delete(0, END)
+    count=0
+
+
+def save_list():    
+    with open('checklist.txt', 'w') as f:        
+        list_tuple = my_listbox.get(0, END)
+        for item in list_tuple:            
+            if item.endswith('\n'):
+                f.write(item)
+            else:
+                f.write(item + "\n")
+
+
+def open_list():    
+    try:
+        with open('checklist.txt', 'r') as f:
+            for line in f:
+                my_listbox.insert(END, line)
+    except:
+        return
+
+
+#Defining layout
+#Defining frames
+input_frame = tkinter.Frame(root, bg=root_color)
+output_frame = tkinter.Frame(root, bg=root_color)
+button_frame = tkinter.Frame(root, bg=root_color)
+input_frame.pack()
+output_frame.pack()
+button_frame.pack()
+
+#Input frame layout
+list_entry = tkinter.Entry(input_frame, width=35, borderwidth=3, font=my_font)
```

Comment:
The code looks good but try to follow PEP8 guidelines. It will help you to write cleaner code.

Reply:
sure